src/interviewer.t

```diff
@@ -217,17 +235,70 @@ warner_interviewer : Person 'interviewer' 'Harried Interviewer' @interview_room
 ;
 
 ++ SpecialTopic 'anime club' ['anime', 'anime club', 'japanese', 'japanese club', 'weeaboo', 'weeb']
-    ""
+    ""//"<q>私はアニメが大好きです~~~!</q>"
```

Comment:
Sounds good, thanks a bunch.

We could probably just leave it out for now without anyone missing much.